### ISSUE TITLE
unable to create or edit product in 8.1.7

### ISSUE THREAD

**Issue by weiserwolf** [PrestaShop/PrestaShop#36717]
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md).
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add attachments

Greetings, I just installed Prestashop 8.1.7 and when trying to create a new product (or edit an existing one), no matter how many information I enter, the Save button is disabled in grey. I don't know what to do. I have PHP 8.1.

```
State version: 66
background.js:2 Migrator lg (to version 3) should migrate: false - up
background.js:2 Migrator Pd (to version 4) should migrate: false - up
background.js:2 Migrator Sh (to version 5) should migrate: false - up
background.js:2 Migrator cp (to version 6) should migrate: false - up
background.js:2 Migrator ig (to version 7) should migrate: false - up
background.js:2 Migrator rg (to version 8) should migrate: false - up
background.js:2 Migrator sg (to version 9) should migrate: false - up
background.js:2 Migrator Uc (to version 10) should migrate: false - up
background.js:2 Migrator Fc (to version 11) should migrate: false - up
background.js:2 Migrator Hc (to version 12) should migrate: false - up
background.js:2 Migrator Wc (to version 13) should migrate: false - up
background.js:2 Migrator Xc (to version 14) should migrate: false - up
background.js:2 Migrator tu (to version 15) should migrate: false - up
background.js:2 Migrator ru (to version 16) should migrate: false - up
background.js:2 Migrator ou (to version 17) should migrate: false - up
background.js:2 Migrator uu (to version 18) should migrate: false - up
background.js:2 Migrator pu (to version 19) should migrate: false - up
background.js:2 Migrator yu (to version 20) should migrate: false - up
background.js:2 Migrator bu (to version 21) should migrate: false - up
background.js:2 Migrator ku (to version 22) should migrate: false - up
background.js:2 Migrator Eu (to version 23) should migrate: false - up
background.js:2 Migrator Du (to version 24) should migrate: false - up
background.js:2 Migrator Pu (to version 25) should migrate: false - up
background.js:2 Migrator Qu (to version 26) should migrate: false - up
background.js:2 Migrator Uu (to version 27) should migrate: false - up
background.js:2 Migrator Fu (to version 28) should migrate: false - up
background.js:2 Migrator qu (to version 29) should migrate: false - up
background.js:2 Migrator Gu (to version 30) should migrate: false - up
background.js:2 Migrator Yu (to version 31) should migrate: false - up
background.js:2 Migrator Zu (to version 32) should migrate: false - up
background.js:2 Migrator ad (to version 33) should migrate: false - up
background.js:2 Migrator ud (to version 34) should migrate: false - up
background.js:2 Migrator pd (to version 35) should migrate: false - up
background.js:2 Migrator yd (to version 36) should migrate: false - up
background.js:2 Migrator bd (to version 37) should migrate: false - up
background.js:2 Migrator xd (to version 38) should migrate: false - up
background.js:2 Migrator jd (to version 39) should migrate: false - up
background.js:2 Migrator _d (to version 40) should migrate: false - up
background.js:2 Migrator Ld (to version 41) should migrate: false - up
background.js:2 Migrator Kd (to version 42) should migrate: false - up
background.js:2 Migrator $d (to version 43) should migrate: false - up
background.js:2 Migrator Jd (to version 44) should migrate: false - up
background.js:2 Migrator nh (to version 45) should migrate: false - up
background.js:2 Migrator sh (to version 46) should migrate: false - up
background.js:2 Migrator fh (to version 47) should migrate: false - up
background.js:2 Migrator Ah (to version 48) should migrate: false - up
background.js:2 Migrator Ih (to version 49) should migrate: false - up
background.js:2 Migrator Dh (to version 50) should migrate: false - up
background.js:2 Migrator Ph (to version 51) should migrate: false - up
background.js:2 Migrator Th (to version 52) should migrate: false - up
background.js:2 Migrator Rh (to version 53) should migrate: false - up
background.js:2 Migrator Fh (to version 54) should migrate: false - up
background.js:2 Migrator Hh (to version 55) should migrate: false - up
background.js:2 Migrator Yh (to version 56) should migrate: false - up
background.js:2 Migrator ep (to version 57) should migrate: false - up
background.js:2 Migrator np (to version 58) should migrate: false - up
background.js:2 Migrator op (to version 59) should migrate: false - up
background.js:2 Migrator gp (to version 60) should migrate: false - up
background.js:2 Migrator Ap (to version 61) should migrate: false - up
background.js:2 Migrator Bp (to version 62) should migrate: false - up
background.js:2 Migrator Tp (to version 63) should migrate: false - up
background.js:2 Migrator Op (to version 64) should migrate: false - up
background.js:2 Migrator $p (to version 65) should migrate: false - up
background.js:2 Migrator eg (to version 66) should migrate: false - up
background.js:2 Uncaught (in promise) Error: Duplicate script ID 'fido2-page-script-registration'
background.js:2 Cannot find menu item with id root
write @ background.js:2
background.js:2 Cannot find menu item with id copy-password
write @ background.js:2
background.js:2 Cannot create item with duplicate id autofill_noop
write @ background.js:2
3background.js:2 Cannot find menu item with id root
write @ background.js:2
background.js:2 Cannot find menu item with id copy-password
write @ background.js:2
background.js:2 Cannot create item with duplicate id autofill_noop
write @ background.js:2
background.js:2 Error while processing message in RuntimeBackground 'triggerAutofillScriptInjection'. Error: Frame with ID 92 was removed.
write @ background.js:2
background.js:2 Cannot find menu item with id root
write @ background.js:2
1018Unchecked runtime.lastError: Could not establish connection. Receiving end does not exist.
background.js:2 Cannot find menu item with id root
write @ background.js:2
background.js:2 Cannot find menu item with id copy-password
write @ background.js:2
background.js:2 Cannot create item with duplicate id autofill_noop
write @ background.js:2
background.js:2 Cannot find menu item with id root
write @ background.js:2
background.js:2 Cannot find menu item with id autofill-card
write @ background.js:2
background.js:2 Cannot create item with duplicate id autofill_noop
write @ background.js:2
background.js:2 Cannot find menu item with id autofill-card
write @ background.js:2
3background.js:2 Cannot find menu item with id root
write @ background.js:2
background.js:2 Cannot find menu item with id copy-username
write @ background.js:2
background.js:2 Cannot find menu item with id root
write @ background.js:2
background.js:2 Cannot find menu item with id autofill-identity
write @ background.js:2
3background.js:2 Cannot find menu item with id root
write @ background.js:2
background.js:2 Error while processing message in RuntimeBackground 'triggerAutofillScriptInjection'. Error: Frame with ID 351 was removed.
write @ background.js:2
3background.js:2 Cannot find menu item with id root
write @ background.js:2
main.bundle.js:1404 JQMIGRATE: Migrate is installed, version 3.3.2
main.bundle.js:98 Failed to initialize PrestaShop component "DeltaQuantityInput". This component doesn't exist.
(anonymous) @ main.bundle.js:98
main.bundle.js:98 Failed to initialize PrestaShop component "ModifyAllShopsCheckbox". This component doesn't exist.
(anonymous) @ main.bundle.js:98
main.bundle.js:98 Failed to initialize PrestaShop component "DisablingSwitch". This component doesn't exist.
(anonymous) @ main.bundle.js:98
main.bundle.js:1404 jQuery.Deferred exception: window.prestashop.component.EntitySearchInput is not a constructor TypeError: window.prestashop.component.EntitySearchInput is not a constructor
    at new Bw (https://clon17.blackfencer.com/Backoffice/themes/new-theme/public/product_edit.bundle.js?8.1.7:1402:71)
    at HTMLDocument.<anonymous> (https://clon17.blackfencer.com/Backoffice/themes/new-theme/public/product_edit.bundle.js?8.1.7:1848:743)
    at o (https://clon17.blackfencer.com/Backoffice/themes/new-theme/public/main.bundle.js:1404:80145)
    at d (https://clon17.blackfencer.com/Backoffice/themes/new-theme/public/main.bundle.js:1404:80469) undefined
M.Deferred.exceptionHook @ main.bundle.js:1404
main.bundle.js:1404 Uncaught TypeError: window.prestashop.component.EntitySearchInput is not a constructor
    at new Bw (product_edit.bundle.js?8.1.7:1402:71)
    at HTMLDocument.<anonymous> (product_edit.bundle.js?8.1.7:1848:743)
    at o (main.bundle.js:1404:80145)
    at d (main.bundle.js:1404:80469)
toolbarContentscript.js:113 {"message":"A listener indicated an asynchronous response by returning true, but the message channel closed before a response was received"}
edit:1 The resource https://clon17.blackfencer.com/Backoffice/themes/new-theme/public/2d8017489da689caedc1.preload..woff2 was preloaded using link preload but not used within a few seconds from the window's load event. Please make sure it has an appropriate `as` value and it is preloaded intentionally.
1025 log entries are not shown.
edit:1 The resource https://clon17.blackfencer.com/Backoffice/themes/new-theme/public/2d8017489da689caedc1.preload..woff2 was preloaded using link preload but not used within a few seconds from the window's load event. Please make sure it has an appropriate `as` value and it is preloaded intentionally.
edit:1 The resource https://clon17.blackfencer.com/Backoffice/themes/new-theme/public/2d8017489da689caedc1.preload..woff2 was preloaded using link preload but not used within a few seconds from the window's load event. Please make sure it has an appropriate `as` value and it is preloaded intentionally.
```__

### Expected behavior

_No response_

### Steps to reproduce

Create or edit a new product, save button in grey.

### PrestaShop version(s) where the bug happened

8.1.7

### PHP version(s) where the bug happened

8.1

### If your bug is related to a module, specify its name and its version

_No response_

### Your company or customer's name goes here (if applicable).

_No response_

**Comment by weiserwolf** [comment]
Just to shed light, it doesn't happen with the Prestashop version 8.1.3 I just installed.

**Comment by jlores** [comment]
Hello,

With other Explorer the error doesn't happen.
I clear cache and ok

**Comment by venditdevs** [comment]
@jlores, @weiserwolf, probably the Product Page V2 is activated. Can you disable the V2 page and try it again?

We had the same problem last month after upgrading to 8.1.7 with V2 active.

You can find this setting under [Advanced Settings] => [New and Experimental Features/Functions].

**Comment by weiserwolf** [comment]
Thank you for all the help, I solved the problem using another browser, in this case Chrome. Brave is giving me a lot of problems lately.

**Comment by monak83** [comment]
Hi, I have the same problem: with page product V2 activated -> button grey and can't save anything!
Note: with PS 8.1.7 clear all was ok, after installed a theme package (with his modules) issue appears!
Tryed to change browsere, clear cache but nothing!
Thnaks for help!
Marco
![cantsave_buttonGREY](https://github.com/user-attachments/assets/dbb2db71-8d2f-42dd-9249-589bda4198fc)

**Comment by Ecommerce-Manager** [comment]
> Hi, I have the same problem: with page product V2 activated -> button grey and can't save anything! Note: with PS 8.1.7 clear all was ok, after installed a theme package (with his modules) issue appears! Tryed to change browsere, clear cache but nothing! Thnaks for help! Marco ![cantsave_buttonGREY](https://private-user-images.githubusercontent.com/8915414/362204947-dbb2db71-8d2f-42dd-9249-589bda4198fc.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjg2NjMwMDgsIm5iZiI6MTcyODY2MjcwOCwicGF0aCI6Ii84OTE1NDE0LzM2MjIwNDk0Ny1kYmIyZGI3MS04ZDJmLTQyZGQtOTI0OS01ODliZGE0MTk4ZmMuanBnP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MTAxMSUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDEwMTFUMTYwNTA4WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ZWY4NWFlODM2MjZiNGIxZDNjZTc4NWE4MDdmNWU0NmNhZDgyZDkyMGMzYzNjNjlhNGRjYmViOTI1ZTc0NTNhMiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.U531xkmmF_pa23ASgN9QE5J4GRSU7pit-zy3MElehp0)

Same thing, error appears only after installing theme/pagebuilder.

**Comment by klapaucius4** [comment]
hi guys,
have you maybe solved this problem? Unfortunately I have the same thing. Interestingly, on the local environment the problem does not appear. Only on DEV / PROD environments.
